understanding/headings-and-labels.html

```diff
@@ -58,190 +56,133 @@
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 2.4.6:</span>Headings and Labels (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 2.4.6:</span>見出し及びラベル (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Headings and <a href="#dfn-label">labels</a> describe topic or purpose.
+                  <p>見出し及び<a href="#dfn-label">ラベル</a>は、主題又は目的を説明している。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>A page's content is described in headings and labels</dd>
+                  <dt>目標</dt>
+                  <dd>ページのコンテンツが、見出し及びラベルによって記述されるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide descriptive headings and labels</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>説明的な見出し及びラベルを提供する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People can orient themselves, especially those with cognitive or visual disabilities.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>特に認知障害又は視覚障害のある人々が、コンテンツを読む方向を自分で把握することができる。</dd>
```

Comment:
orient oneself
自分の位置［立場］を知る［確認する］
<https://eow.alc.co.jp/search?q=orient+themselves>
ですので、例えばこうでしょうか？

```suggestion
                  <dd>特に認知障害又は視覚障害のある人々が、ページ上の自分の位置を確認することができる。</dd>
```

Reply:
@momdo なるほどです。orient onself でひとつのイディオムなのですね。いただいた修正提案のほうがよいと思います。(ありがとうございます...！)